emailer.py

```diff
@@ -12,6 +10,8 @@
 import rollbar
 import rollbar.contrib.flask
 import sha
+import smtplib
```

Comment:
Do we need to update `requirements.txt` now? (Dropping `Envelopes` and adding whatever package has `smtplib`  )

Reply:
Ah, looks like `smtplib` is part of the standard library so probably don't need a new package, but could still drop envelopes?

Reply:
smptlib is a default python library. I could remove Envelopes for sure but I shouldn't have to add anything.